### PR TITLE
Database connection employees register form

### PR DESCRIPTION
**Pull request by DiegoMoralesBravo**
# Description

What did you implement/Why did you implement this:

- Employee register form created
- Database connection between register employee form and employee table

## Test

* Move to /employees/new path or click in the add employee button in the /employee path
* Fill the form and submit
* Check the employee's table for the employee created

## Screenshots

image

### CODE REVIEW

**Review by AangelF — APPROVED** [review]

**Review by DiegoCoronaBarragan — APPROVED** [review]
LGTM

**Review by Yaxkin121 — APPROVED** [review]
LGTM

**Review by EverardoAlcarazBaltazar — APPROVED** [review]
LGTM

**Review by oscaregr — APPROVED** [review]

File: app/assets/stylesheets/pages/_crud_employee.css.scss [hunk 1]
```diff
@@ -36,6 +36,12 @@
 
   .add-text-container {
     margin-left: 17px;
+
+    a {
+      text-decoration: none;
+      color: $font-color-gray;
```
Comment (juperameza) [inline app/assets/stylesheets/pages/_crud_employee.css.scss:42]: Alphabetically order this

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class EmployeesController < ApplicationController
+
```
Comment (juperameza) [inline app/controllers/employees_controller.rb:2]: At this moment everyone can access this page shouldn't you protect it by adding `   before_action :authenticate_user!` or whatever session login that you're using

File: app/views/employees/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          <%= form.label :private_number %><br>
+          <%= form.text_field :private_number %>
+        </div>
+        <div class="button-container">
+          <%= form.submit %>
+        </div>
+      <% end %>
+    </section>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/employees/new.html.erb:34]: Add a blank line after this

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by dagp-ucol — APPROVED** [review]

**Review by juperameza — APPROVED** [review]
Lgtm

**Pull request merged**